packages/contracts/contracts/L1/IValidatorManager.sol

```diff
@@ -0,0 +1,255 @@
+     */
+    event Slashed(address indexed loser, address indexed winner, uint128 amount);
+
+    /**
+     * @notice Registers as a validator with assets at least MIN_REGISTER_AMOUNT. The validator with
+     *         assets more than MIN_START_AMOUNT can be started at the same time.
+     *
+     * @param assets                  The amount of assets to self-delegate.
+     * @param commissionRate          The commission rate the validator sets.
+     * @param commissionMaxChangeRate The max change rate of commission the validator sets.
+     */
+    function registerValidator(
+        uint128 assets,
+        uint8 commissionRate,
+        uint8 commissionMaxChangeRate
+    ) external;
+
+    /**
+     * @notice Starts a validator and adds the validator to weight tree. To submit outputs, the
+     *         validator should start.
+     */
+    function startValidator() external;
+
+    /**
+     * @notice Handles some essential actions such as reward distribution, jail handling, next
+     *         priority validator selection after output submission. This function can only be
+     *         called by L2OutputOracle.
+     *
+     * @param outputIndex Index of the L2 checkpoint output submitted.
+     */
+    function afterSubmitL2Output(uint256 outputIndex) external;
+
+    /**
+     * @notice Changes the commission rate of a validator. An inactive validator cannot change it,
+     *         and a validator can change it after COMMISION_RATE_MIN_CHANGE_SECONDS elapsed since
+     *         the last changed time. Also, the validator can only make changes within the
+     *         commissionMaxChangeRate that the validator set initially.
+     *
+     * @param newCommissionRate The new commission rate to apply.
+     */
+    function changeCommissionRate(uint8 newCommissionRate) external;
+
+    /**
+     * @notice Attempts to unjail a validator. Only the validator who wants to unjail can call.
+     */
+    function tryUnjail() external;
+
+    /**
+     * @notice Updates the weight tree of the validator.
+     *
+     * @param validator Address of the validator.
+     * @param tryRemove Flag to try remove the validator from weight tree.
+     */
+    function updateValidatorTree(address validator, bool tryRemove) external;
+
+    /**
+     * @notice Checks the eligibility to submit L2 checkpoint output during output submission.
+     *         Note that only the validator whose status is CAN_SUBMIT_OUTPUT can submit output.
+     *         This function can only be called by L2OutputOracle during output submission.
+     *
+     * @param validator Address of the output submitter.
+     */
+    function checkSubmissionEligibility(address validator) external view;
+
+    /**
+     * @notice Determines who can submit the L2 checkpoint output for the current round.
+     *
+     * @return Address of the validator who can submit the L2 checkpoint output for the current
+     *         round.
+     */
```

Comment:
```suggestion
     * @notice Determines who can submit the L2 checkpoint output for the next round.
     *
     * @return Address of the validator who can submit the L2 checkpoint output for the next
     *         round.
     */
```

Reply:
Actually, it returns the address according to the round (priority or public), so `current` is more appropriate.